Tests/FirebaseTests/Tests/DynamicLinks/DynamicLinksEndpointsTests.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+@testable import Firebase
+@testable import Vapor
+
+final class DynamicLinksEndpointsTests: XCTestCase {
+    
+    func test_shortinkEndpoint_value() throws {
+        // Given
+        let expectedEnpoint = "https://firebasedynamiclinks.googleapis.com/v1/shortLinks"
+        
+        // When
+        let endpoint = FirebaseAPIEndpoint.DynamicLinks.shortLinks.endpoint
+        
+        // Then
+        XCTAssertEqual(endpoint, expectedEnpoint)
+    }
+    
+    static var allTests = [
+        ("test_shortinkEndpoint_value", test_shortinkEndpoint_value),
```

Comment:
Naming will be addressed by the change requested above.

Reply:
https://github.com/h2glab/firebase-provider/pull/3/commits/d6d756dc72192a0d4d003929589046a443df4802